connectors/google/google-sheets/element-templates/google-sheets-connector.json

```
@@ -427,15 +423,44 @@
       }
     },
     {
-      "label": "Column index",
+      "id": "columnIndexRepresentation",
+      "label": "Index representation",
```

Condsider following wording

```suggestion
      "label": "Index format",
```

---

done